docker-deploy/frontend/docker-entrypoint.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
```

Comment:
Put the license notice to the beginning of all code files following this template:
"
    <one line to give the program's name and a brief idea of what it does.>
    Copyright (C) <year>  <name of author>

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU Affero General Public License as
    published by the Free Software Foundation, either version 3 of the
    License, or (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU Affero General Public License for more details.

    You should have received a copy of the GNU Affero General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.
"

Reply:
This template could be found easily at the end of LICENSE file.

Reply:
Thanks, I added the license